### PR TITLE
feature/se-48 & se-52

### LINKED ISSUE #48: Company Course Management - Front-end Creation
image

### PR DESCRIPTION
**Pull request by Ara-Ara-ASid-kun**
Closes https://github.com/SE2-WS23/documentation/issues/48 & https://github.com/SE2-WS23/documentation/issues/52

Redesigned the styling mostly according to JobsProfile page

Used ListItemText instead of Typography in components file as per previous request 

Have a hungry Konata to warm your hearts
Konata Sleepy

### CODE REVIEW

File: src/pages/CompanyTrainingOverview/components/CourseListItem.jsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+          flexDirection: 'column',
+          alignItems: 'flex-start',
+          justifyContent: 'flex-start',
+        }}
+      >
+        <ListItemText
+          primary={courseCategory}
+          primaryTypographyProps={{
+            variant: 'body1',
+            sx: {
```
Comment (Terminaluser3) [inline src/pages/CompanyTrainingOverview/components/CourseListItem.jsx:24]: Lower resolutions cause the text to overflow here. You could add something like this to stop that from happening 
```suggestion
            sx: {
            wordWrap: "break-word",
            width: "70%",
```
. You could also use a grid.

**Comment by Terminaluser3** [comment]
Also, please correct me if I'm wrong, but aren't the component names switched? "CompanyTrainingOverview"'s component is called "CourseListItem" , and vice versa.

File: src/pages/CompanyCourseManagement/components/TrainingListItem.jsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      <ListItem
+        style={{
+          height: '150px',
+          background: theme.palette.primary.light,
+          display: 'flex',
+          flexDirection: 'column',
+          alignItems: 'flex-start',
+          justifyContent: 'flex-start',
+        }}
+      >
```
Comment (Markus799) [inline src/pages/CompanyCourseManagement/components/TrainingListItem.jsx:19]: image

File: src/pages/CompanyTrainingOverview/components/CourseListItem.jsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+import PropTypes from 'prop-types';
+
+function CourseListItem({ courseName, courseCategory, trainingNumber }) {
+  const theme = useTheme();
+
+  return (
+    <Grid item xs={6}>
+      <ListItem
+        style={{
+          height: '150px',
```
Comment (Markus799) [inline src/pages/CompanyTrainingOverview/components/CourseListItem.jsx:12]:
```suggestion
          height: '100px',
```

use a height of 100px to have it identical to jobProfile

File: src/pages/CompanyCourseManagement/components/TrainingListItem.jsx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+import PropTypes from 'prop-types';
+
+function TrainingListItem({ trainingName, trainingCategory }) {
+  const theme = useTheme();
+
+  return (
+    <Grid item xs={6}>
+      <ListItem
+        style={{
+          height: '150px',
```
Comment (Markus799) [inline src/pages/CompanyCourseManagement/components/TrainingListItem.jsx:12]:
```suggestion
          height: '100px',
```

use a height of 100px to have it identical to jobProfile

File: src/pages/CompanyCourseManagement/CompanyCourseManagementPage.jsx [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+      <Button
+        variant='contained'
+        color='primary'
+        size='large'
+        sx={{
+          margin: '116px 0 0px 116px',
+        }}
+        onClick={handleAddTrainingClick}
+      >
+        ADD TRAINING
```
Comment (Markus799) [inline src/pages/CompanyCourseManagement/CompanyCourseManagementPage.jsx:27]:
```suggestion
        ADD COURSE
```

File: src/pages/CompanyTrainingOverview/CompanyTrainingsOverviewPage.jsx [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+      <Button
+        variant='contained'
+        color='primary'
+        size='large'
+        sx={{
+          margin: '116px 0 0px 116px',
+        }}
+        onClick={handleAddCourseClick}
+      >
+        ADD COURSE
```
Comment (Markus799) [inline src/pages/CompanyTrainingOverview/CompanyTrainingsOverviewPage.jsx:27]:
```suggestion
        ADD TRAINING
```

**Comment by Chrissydrx** [comment]
Wrong naming of the branch. Please rename the branch and open a new pull request

**Pull request closed without merging**